src/context.cpp

```diff
@@ -87,8 +87,9 @@ namespace Sass {
 
   {
 
-    // add cwd to include paths
-    include_paths.push_back(CWD);
+    // Sass 3.4: The current working directory will no longer be placed onto the Sass load path by default.
```

Comment:
I understand, however it's a breaking change and 3.4 is stable. The ship has sailed on breaking changes in 3.4. Back ports are only happening for small Sass fixes.